src/main/java/org/opentripplanner/transit/model/timetable/TripTimes.java

```diff
@@ -541,6 +542,76 @@ public boolean adjustTimesBeforeAlways(int firstUpdatedIndex) {
     return hasAdjustedTimes;
   }
 
+  /**
+   * Interpolate the times for CANCELLED stops in between regular stops to ensure internal time
+   * representations are increasing for these stops. This will not interpolate terminal
+   * cancellations, as those can be handled by backward and forward propagations. Note: This is
+   * currently for GTFS-RT only, since SIRI requires times for all stops.
```

Comment:
```suggestion
   * Interpolate the times for CANCELLED stops in between regular stops to ensure internal time
   * representations are increasing for these stops. This will not interpolate terminal
   * cancellations, as those can be handled by backward and forward propagations. Note: This is
   * currently for GTFS-RT only, since SIRI requires times for all stops and there we _want_ to reject stops where the 
   * times for cancelled stops are missing or are non-increasing.
```